modules/ROOT/pages/_partials/sync/getting-started-openshift.adoc

```diff
@@ -0,0 +1,73 @@
+= Prerequisites
+* [x] You have Docker installed.
+* [x] You have access to an OpenShift cluster with the Service Catalog.
+* [x] You have completed the server getting started guide.
+
+= Overview
+
+To connect your Data Sync server and client to other mobile services, you need to run your application in OpenShift.  Data Sync provides a service catalog item to help with this.
+
+Data Sync requires your server application to be packaged as a Docker formatted container and published to a public respository such as link:https://hub.docker.com/[Docker hub]. 
+
+= Building and publishing the server container
+
+To build a server into a container, create a `Dockerfile` in the project's directory.  This container will need to include your server source code, its dependencies, and be configured to execute your server.
+
+As an example:
+
+[source,dockerfile]
+----
```

Comment:
Why not use s2i node that will work here and it is fully supported way to run node.js app on OpenShift. 
There are also buckarest.gold images but that is hard to get by default on openshift.